apps/mobile/src/state/tradeChecklist/atoms.ts

```diff
@@ -0,0 +1,4 @@
+import {atom} from 'jotai'
+import {type Uuid} from '@vexl-next/domain/dist/utility/Uuid.brand'
+
+export const tradeChecklistUuidAtom = atom<Uuid | undefined>(undefined)
```

Comment:
As written above. Let's remove this from here and put this into tradeChecklist component.